src/opustrainer/alignments.py

```diff
@@ -2,12 +2,12 @@
 from opustrainer.types import Pair, TokenList
 
 
-def parse_alignments(pairs:str, src_tokens:Optional[TokenList]=None, trg_tokens:Optional[TokenList]=None) -> List[Pair]:
+def parse_alignments(input_pairs:str, src_tokens:Optional[TokenList]=None, trg_tokens:Optional[TokenList]=None) -> List[Pair]:
     """Parses `1-2 3-4` into `[Pair(src=1,trg=2), Pair(src=3,trg=4)]`. If `src_tokens` and
     `trg_tokens` are supplied, it will also check that the indices are not out of bounds."""
-    pairs = [
+    pairs: List[Pair] = [
```

Comment:
The next line is pretty explicit about the type already… If this were C++, wouldn't you have used `auto` here?

Reply:
The Linter doesn't get it, that's why i added it ; (